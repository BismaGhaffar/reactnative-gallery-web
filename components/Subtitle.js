import styled from 'styled-components'

const Subtitle = styled.h3`
  position: relative;
  z-index: 100;
  text-align: center;
  @media (min-width: 481px) {
    font-size: 20px;
  }
  @media (min-width: 769px) {
    font-size: 25px;
  }
  @media (max-width: 769px) {
    display: ${props => (props.hidexs ? 'none' : 'block')};
  }
`

export default Subtitle
