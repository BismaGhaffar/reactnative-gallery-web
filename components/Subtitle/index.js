import styled from 'styled-components'

const Subtitle = styled.h3`
  font-size: 1em;
  text-align: center;
  color: #000;
`

export default Subtitle
