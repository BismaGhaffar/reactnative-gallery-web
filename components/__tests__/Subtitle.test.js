import React from 'react'
import renderer from 'react-test-renderer'
import Subtitle from '../Subtitle'

it('Subtitle can be created', () => {
  const comp = renderer.create(<Subtitle />)
  expect(comp).toBeDefined()
})

it('<Subtitle /> toMatchSnapshot', () => {
  const tree = renderer.create(<Subtitle />).toJSON()
  expect(tree).toMatchSnapshot()
})
