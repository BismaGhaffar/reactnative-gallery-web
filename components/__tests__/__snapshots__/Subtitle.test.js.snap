// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Subtitle /> toMatchSnapshot 1`] = `
<h3
  className="Subtitle-s1pqjucr-0 kyHEyY"
/>
`;

exports[`<Subtitle /> toMatchSnapshot 2 1`] = `
<h3
  className="Subtitle-s1pqjucr-0 hQpFcq"
/>
`;
